quickstarts/notification-channels/servicenow-notifications/config.yml

```diff
@@ -0,0 +1,34 @@
+name: servicenow-notifications
+
+title: ServiceNow Notifications
+
+description:  |
+  
+  ServiceNow is a software that helps companies manage digital workflows for enterprise operations.
+
+  Check out our documentation to set up a ServiceNow notification channel and provide fast and consistent ways for the right personnel to be notified about incidents
+
+summary: |
+  Send your New Relic alerts to ServiceNow
+
+logo: logo.png
+
+level: New Relic
+
+authors:
+  - Matan Moser (New Relic)
+  - Ismail Azam (New Relic)
+
+keywords:
+  - servicenow
+  - notifications
+  - AIOps
+  - Incident Intelligence
+  - Destinations
+
+documentation:
```

Comment:
Won't this add a tile to the data source tab? Which I thought we agreed would confuse the user and the idea was to just link to these doc URLS from within the markdown description.